src/main/java/com/project/workingtime/CheckController.java

```diff
@@ -0,0 +1,24 @@
+package com.project.workingtime;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@RestController
+public class CheckController {
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
+    LocalDateTime now = LocalDateTime.now();
```

Comment:
The caller should not be able to directly access the fields `dtf` and `now`. They should be declared private.